tests/gdx-tests/src/com/badlogic/gdx/tests/gwt/GwtTestWrapper.java

```diff
@@ -418,6 +418,11 @@ public boolean isCatchMenuKey () {
 			return input.isCatchMenuKey();
 		}
 
+		@Override
+		public void setCatchKey(int keycode, boolean catchKey) {
+
```

Comment:
For consistency's sake I'd change this to
````java
input.setCatchKey(keycode, catchKey);
````
Since the other methods in `GwtTestWrapper` are doing the same.